### DISCUSSION TITLE
Last Seen Room Temperature Datetime...

### DISCUSSION THREAD

**Discussion by MBRetrofit** [jmcollin78/versatile_thermostat#674] [General]
As the summer on my place was very long I missed the latest updates and although I spend some time I did not find the instruction for this Last Seen Room Temperature Datetime.

Please point me from where this is explained.

**Comment by jmcollin78** [comment]
Hello @MBRetrofit ,

I'm on the way to rewrite the documentation so it is not fully available now. A Last seen sensor is given by ZigbeeToMqtt addon on certain temperature sensor device. If you enable this entity you will have a sensor like this:
<img width="259" alt="Capture d’écran 2024-12-01 à 22 14 34" src="https://github.com/user-attachments/assets/050ff656-bf69-406c-92d9-c6c5f905e005">

If you have one, you can give it in the main attributes of the VTherm configuration. It will help to prevent the safety mode.

**Reply by MBRetrofit** [reply]
Ah, nice. I will check my Xiaomi sensora.

VTherm will simulate a temperature of this Last Seen will be too long?

**Reply by jmcollin78** [reply]
No. VTherm will consider that the t° sensor is alive when the last seen sensor is updated and avoid then to put it into safety mode.
VTherm keeps the last t° received as the current room temperature.

**Reply by MBRetrofit** [reply]
For a specific timeframe? As if not, the safety mode will be cancelled

**Reply by jmcollin78** [reply]
As long as the thermometer is alive (changing temperature or changing last seen)
